### PR TITLE
split project intro sub-projects

### PR DESCRIPTION
**Pull request by willyborankin**
gradle build for multi-project settings

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,200 +13,145 @@
...
     }
-}
 
-idea {
-    module {
-        testSourceDirs += project.sourceSets.integrationTest.java.srcDirs
-        testSourceDirs += project.sourceSets.integrationTest.resources.srcDirs
-    }
-}
+    distributions {
```
Comment (ivanyu) [inline build.gradle:79]: Looks duplicate

**Review by ivanyu — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (willyborankin) [reply build.gradle]: true

**Review by ivanyu — APPROVED** [review]
LGTM.
Do you plan to squash?

**Comment by willyborankin** [comment]
@ivanyu squashed

**Pull request merged**